dsn-2018/bare_conf.tex

```diff
@@ -0,0 +1,1201 @@
+inductively building full collisions from reduced round variants of MD4. We
+propose these techniques as a mechanism for measuring hash trustworthiness and
+discuss potential applications to real-world systems.
+\end{abstract}
+
+
+
+
+
+
+% no keywords
+% For peer review papers, you can put extra information on the cover
+% page as needed:
+% \ifCLASSOPTIONpeerreview
+% \begin{center} \bfseries EDICS Category: 3-BBND \end{center}
+% \fi
+%
+% For peerreview papers, this IEEEtran command inserts a page break and
+% creates the second title. It will be ignored for other modes.
+\IEEEpeerreviewmaketitle
+
+\section{Introduction}
+
+Cryptographic hash functions form the core of many protocols. From file
+integrity checks to verify long term storage of data, to cache invalidation
+techniques, and use as a building block in network protocols such as Kerberos
+and TLS, this class of functions necessarily has strong guarantees about
+properties of its members. There are three basic properties hash
+functions must have to be considered cryptographically secure:
+\begin{itemize}
+    \item \textbf{Preimage Resistance}: It should be computationally hard to find
+        the inverse of a hash function.
+    \item \textbf{Second Preimage Resistance}: Given an input block, it should be
+        computationally hard to find a second block which hashes to the
+        same value as the first block.
+    \item \textbf{Collision Resistance}: It should be computationally hard to find two
+        blocks which hash to the same value.
+\end{itemize}
+Note that a second preimage is necessarily a collision and is also a stronger
+result in practice.
+
+From an attacker's perspective, finding a preimage or second preimage has been
+difficult. To the author's knowledge, the current bound for a preimage attack
+in MD5 is $2^{123.4}$ by Y. Sasaki and K. Aoki \cite{SasakiPreimage}. Further,
+while J. Kelsey and B. Schneier proposed a method for finding second preimages
+in less than $2^n$, we note that this requires rather long messages
+\cite{SchneierSecondPreimage}. However, collision attacks are well within reach
+for adversaries, and well past only theoretical attacks. The work of
+H. Dobbertin \cite{Dobbertin1998}, X. Wang \cite{cryptoeprint:2004:199},
+M. Schl{\"a}ffer \cite{Schlaffer2006} and others demonstrate the ease with which
+collisions can be found.
+
+From a cryptographic perspective, the existence of a feasible collision attack
+breaks the requisite properties, thus deprecating the function's use. In some
+instances however, there continues to be widespread use of deprecated hash
+functions. One widespread example is the continued use by Git of SHA-1, despite
+M. Stevens et al. publishing a full SHA-1 collision in February of 2017
+\cite{cryptoeprint:2017:190}. However, this attack had a bound of $2^{63.1}$ --
+faster than the theoretical $2^{80}$ but still requiring significant compute
```

Comment:
is "significant compute resources" a technical term, because otherwise I would say "significant **computing** resources" or "significant resources **to compute**"

Reply:
added "to find"